test/runtests.jl

```diff
@@ -178,10 +178,10 @@ anorexia = CSV.read(joinpath(glm_datadir, "anorexia.csv"))
     @test isapprox(aicc(gm6), 490.8823884513153)
     @test isapprox(bic(gm6), 501.35662813730465)
     @test isapprox(coef(gm6),
-        [45.674043486911685, -0.5655388496390964, 4.097065528072901, 8.660128180991693])
+        [49.7711090, -0.5655388, -4.0970655, 4.5630627])
```

Comment:
For reference, these changes are needed because CSV now sorts levels in lexicographic order. So the coefficients correspond to different contrasts than before.